### ISSUE TITLE
FEAT: Possibility to display field comments in the list

### ISSUE THREAD

**Issue by Twor** [beekeeper-studio/beekeeper-studio#722]
When I open a data table, there is only data information, no corresponding comments. And I can only see the comments by show `full columns from table_name;`

**Comment by rathboma** [comment]
Yeah we like this too and have been thinking about a way to include it. I'm not sure it should go in the data view, but certainly the properties view.
